lib/svgo/js2svg.js

```diff
@@ -64,15 +66,26 @@ function JS2SVG(config) {
     this.config.indent = '    ';
   }
 
+  if (typeof this.config.eol === 'string' && this.config.eol) {
+    const configuredEOL = this.config.eol.toLowerCase();
+    if (configuredEOL === 'lf') {
+      this.eol = '\n';
+    } else if (configuredEOL === 'crlf') {
+      this.eol = '\r\n';
+    } else {
+      this.eol = this.config.eol;
+    }
+  }
```

Comment:
This conversion is no longer necessary since we pass newlines directly

Reply:
I'm concerned that if we change the configuration option to only accept the actual line break characters themselves as opposed to supplying `--eol crlf` or `--eol lf`, line break characters may have unintended consequences in certain commands or configuration files. That's why I originally made the option take `crlf` or `lf`, similar to how other tools and configurations handle such options, such as git or editorconfig.

Reply:
I see. Ok, we better avoid config complexity and stick with `lf`/`crlf` enum without else case. And in defaults please derive `lf`/`crlf` from os.eol.

Reply:
Addressed in 21bb65f.